llvm/lib/IR/Verifier.cpp

```diff
@@ -723,6 +724,16 @@ static void forEachUser(const Value *User,
   }
 }
 
+void Verifier::visitTargetExtType(TargetExtType *TTy) {
```

Comment:
I don't think the verifier has a good way to visit all used types, does it? This function is only called from visitGlobalVariable and visitAllocaInst which is clearly not exhaustive(!).

Reply:
maybe you could dig around in the type cache, but that's ugly

Reply:
Could that cache ever contain types that are no longer reachable from the Module? I wouldn't want to verify those.